inference_segmentation.py

```diff
@@ -325,17 +346,22 @@ def main(params):
 
     h, w = [side for side in dm.inference_dataset.src.shape]
 
+    # window_spline_2d = None 
     if auto_batch_size and device.type != "cpu":
-        batch_size = auto_batch_size_finder(datamodule=dm,
+        window_spline_2d = auto_batch_size_finder(datamodule=dm,
```

Comment:
Again, I'm not sure I understand why the auto batch size finder returns a window spline which is only affected by patch size. I'd have to test this to make sure.